packages/utils/src/fhir-converter/README.md

```diff
@@ -0,0 +1,18 @@
+# Testing Flow
+
+Each of these scripts is used by passing in the file you want to compare:
+
+`npm run compare-total-resource-counts "$FILE_LOCATION"`
+`npm run compare-resource-counts-per-file "$FILE_LOCATION"`
+`npm run compare-total-resource-count-post-hapi "$FILE_LOCATION"`
```

Comment:
nit: `post-hapi` -> `post-fhir-insert`, so we don't have to change these names if we migrate FHIR backends

ditto everywhere the file name is used